copulas/bivariate/base.py

```diff
@@ -76,12 +77,19 @@ def fit(self, U, V):
         self.tau = stats.kendalltau(self.U, self.V)[0]
         self.theta = self.tau_to_theta()
 
-    def get_params(self):
-        return {'tau': self.tau, 'theta': self.theta}
+    def to_dict(self):
+        return {
+            'copula_type': self.copula_type.name,
+            'theta': self.theta,
+            'tau': self.tau
+        }
 
-    def set_params(self, **kwargs):
-        for key, value in kwargs.items():
-            setattr(self, key, value)
+    @classmethod
+    def from_dict(cls, **kwargs):
```

Comment:
This should expect a variable called `copula_dict` or similar, which should be a dict, instead of kwargs.